chromium_src/base/metrics/histogram_functions.h

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_BASE_METRICS_HISTOGRAM_FUNCTIONS_H_
+#define BRAVE_CHROMIUM_SRC_BASE_METRICS_HISTOGRAM_FUNCTIONS_H_
+
+#include <stdint.h>
+
+#define uintmax_t intmax_t
```

Comment:
This seems dangerous to me: `base/metrics/histogram_functions.h` pulls in other includes (that also pull in includes). We don't know what we may be overriding here.

Reply:
Spoke with Max and removed this override and replaced it with an explicit patch here 8ca8a5f158ce83d258779f06fd6d4738ef68fdb3.